tools/tpm2_nvread.c

```diff
@@ -165,6 +165,11 @@ static bool init(int argc, char *argv[], tpm_nvread_ctx *ctx) {
                         optarg);
                 return false;
             }
+            if (ctx->size_to_read > 1024) {
```

Comment:
@williamcroberts correct, just need to test the nv index size prior with nv read public to avoid nv wear out.